azerothcore/azerothcore-wotlk
# fix(DB/Conditions): Fix Midsummer Bonfire gossip menu option conditions

## Changes Proposed:

This PR proposes changes to:

## Issues Addressed:

- Closes 

## SOURCE:

The changes have been validated through:

## Tests Performed:

This PR has been:


## How to Test the Changes:



.event start 1

- visit Bonfire of opposing faction
- complete quest to desecrate the fire
- re-open gossip menu
- gossip option to desecrate the fire again (without quest) should be available now

-- Alliance Bonfires
.go ga 76304

-- Horde Bonfires
.go ga 76300
.go ga 76305
.go ga 76309
.go ga 76310
.go ga 76317
.go ga 76318
.go ga 76320
.go ga 76322
.go ga 76324
.go ga 76325
.go ga 76327
.go ga 76328
.go ga 76331
.go ga 76332
.go ga 76333
.go ga 76334
.go ga 76336
.go ga 76337
.go ga 76341
.go ga 76346
.go ga 76348
.go ga 76351
.go ga 76353
.go ga 76355
.go ga 76357
.go ga 76358
.go ga 76360
.go ga 76361

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
